editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -867,6 +867,8 @@ void EditorAssetLibrary::_search(int p_page) {
 	}
 	args += String() + "sort=" + sort_key[sort->get_selected()];
 
+	args += "&godot_version=3.0";
```

Comment:
can we grab version number from somewhere, instead of hard code?

Reply:
I looking for possible solution...